src/main/kotlin/model/ManagePOMCases.kt

```diff
@@ -0,0 +1,160 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.*
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class ManagePOMCases private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var ldu: Person
+    lateinit var pom: Person
+    lateinit var homd: Person
+    lateinit var allocationManager: Container
+    lateinit var redis: Container
+    lateinit var activeJob: Container
+    lateinit var managePOMCasesAPI: Container
+    lateinit var db: Container
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "Manage POM Cases",
+        "A service for handling the handover of offenders from prison to probation"
+      )
+
+      ldu = model.addPerson(
+        "Local Delivery Unit",
+        "A Community/Probation team handling offenders who hail from a defined geographical area"
+      )
+
+      pom = model.addPerson(
+        "Prison Offender Manager (POM)",
+        "A prison staff member allocated to the prisoner"
+      )
+
+      homd = model.addPerson(
+        "Head Of Offender Management Delivery (HOMD)",
+        "A senior staff member who manages the prison's POM team"
+      )
+
+      allocationManager = system.addContainer(
+        "Offender Management Allocation Manager",
+        "Main application code - holds the UI and primary business logic",
+        "Ruby on Rails"
+      ).apply {
+        setUrl("https://github.com/ministryofjustice/offender-management-allocation-manager")
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      db = system.addContainer(
+        "MPC database",
+        "Database holding data mastered by MPC",
+        "AWS RDS PostgreSQL"
+      ).apply {
+          Tags.DATABASE.addTo(this)
+          CloudPlatform.rds.add(this)
+      }
+
+      managePOMCasesAPI = system.addContainer(
+        "manage POM Cases API",
```

Comment:
💭 Is this the same as `allocationManager`? The container (deployable) could serve the purpose of a web app, REST API and other functions, we don't have to separate them.

Reply:
The API is exposed from the same "block" of code which exposes the UI.

The service is written in RoR app which natively encourages a service orientated approach but within a single monolithic application - I'm looking to expose the internal breakdown of this monolith into it's constituent components - help/guidance on this very welcome!!